examples/backends/vllm/mm_router_worker/mm_processor.py

```
@@ -0,0 +1,379 @@
+        return tokenizer.encode(prompt), None
+
+
+def _compute_tokens_per_image(
+    processor_output: dict, processor: Any
+) -> list[int]:
+    """Compute the number of visual tokens for each image from processor output."""
+    grid_thw = processor_output.get("image_grid_thw")
+    if grid_thw is None:
+        raise ValueError(
+            "image_grid_thw not found in processor output"
+        )
```

Ruff (0.15.0)

[warning] Avoid specifying long messages outside the exception class

(TRY003)